cisco_aci/datadog_checks/cisco_aci/data/conf.yaml.example

```diff
@@ -1,49 +1,97 @@
+init_config:
+
+  ## @param min_collection_interval - integer - optional - default: 180
+  ## This check makes a lot of API calls
+  ## it could sometimes help to add a minimum collection interval
+  #
+  #  min_collection_interval: 180
+
 instances:
-    - aci_url: localhost
-      # This check makes a lot of API calls
-      # it could sometimes help to add a minimum collection interval
-      # min_collection_interval: 180
-      # Add multiple urls if there are multiple controllers,
-      # the check will balance between them so it doesn't make too many requests
-      aci_urls:
-        - localhost:1234
-        - localhost:5555
-      username: datadog
-
-      # Authentication can use either a password or a certificate.
-      pwd: datadog
-
-      # Certificate Authentication
-      # When adding a cert to the ACI, it requires a name.
-      # cert_name: datadog
-      # Add the whole cert in the config file
-      # cert_key: >
-      #   -----BEGIN PRIVATE KEY-----
-      #   MIICdQIBADANBgkqhkiG9w0BAQEFAASCAl8wggJbAgEAAoGBAOLJKRals8tGoy7K
-      #   ljG6/hMcoe16W6MPn47Q601ttoFkMoSJZ1Jos6nxn32KXfG6hCiB0bmf1iyZtaMa
-      #   idae/ceT7ZNGvqcVffpDianq9r08hClhnU8mTojl38fsvHf//yqZNzn1ZUcLsY9e
-      #   wG6wl7CsbWCafxaw+PfaCB1uWlnhAgMBAAECgYAI+tQgrHEBFIvzl1v5HiFfWlvj
-      #   DlxAiabUvdsDVtvKJdCGRPaNYc3zZbjd/LOZlbwT6ogGZJjTbUau7acVk3gS8uKl
-      #   ydWWODSuxVYxY8Poxt9SIksOAk5WmtMgIg2bTltTb8z3AWAT3qZrHth03la5Zbix
-      #   ynEngzyj1+ND7YwQAQJBAP00t8/1aqub+rfza+Ddd8OYSMARFH22oxgy2W1O+Gwc
-      #   Y8Gn3z6TkadfhPxFaUPnBPx8wm3mN+XeSB1nf0KCAWECQQDlSc7jQ/Ps5rxcoekB
-      #   ldB+VmuR8TfcWdrWSOdHUiLyoJoj+Z7yfrf70gONPP9tUnwX6MYdT8YwzHK34aWv
-      #   8KiBAkBHddlql5jDVgIsaEbJ77cdPJ1Ll4Zw9FqTOcajUuZJnLmKrhYTUxKIaize
-      #   BbjvsQN3Pr6gxZiBB3rS0aLY4lgBAkApsH3ZfKWBUYK2JQpEq4S5M+VjJ8TMX9oW
-      #   VDMZGKoaC3F7UQvBc6DoPItAxvJ6YiEGB+Ddu3+Bp+rD3FdP4iYBAkBh17O56A/f
-      #   QX49RjRCRIT0w4nvZ3ph9gHEe50E4+Ky5CLQNOPLD/RbBXSEzez8cGysVvzDO3DZ
-      #   /iN4a8gloY3d
-      #   -----END PRIVATE KEY-----
-      # Or, keep the cert in a file and add the file
-      # cert_key_path: /etc/datadog-agent/conf.d/cisco_aci.d/cert.key
-      # If there is a password to the key, add it here
-      # cert_key_password: datadog
-
-      timeout: 15
-      ssl_verify: True
-      tags: []
-      tenant:
-        - WebApp
-        - Database
-        - Datadog
+
+  ## @param aci_url - string - required
+  ## Url to query to gather metrics.
+  #
+  - aci_url: localhost
+
+  ## @param username - string - required
+  ## Authentication can use either a user auth or a certificate.
+  ## If using the user auth, enter in this parameter the associated username.
+  #
+    username: datadog
+
+  ## @param pwd - string - required
+  ## Authentication can use either a user auth or a certificate.
+  ## If using the user auth, enter in this parameter the associated password.
+  #
+    pwd: datadog
+
+  ## @param tenant - list of strings - required
```

Comment:
what is the `tenant` parameter, is it required?

Reply:
optional: list of tenants to collect data from